### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by CarloHD**
Adding "urpredditodicittadinanza.lavoro.gov.it" site and set bg-color to lwc var (previuosly not setted)

Fix google drive white bg-color on "empty the trash" bar on trash foder, set a var color and change bg-color on "New" button with new var

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5221,7 +5221,10 @@ div[role="button"][aria-disabled="true"] > div {
     filter: invert(93.7%) hue-rotate(180deg) contrast(90.6%) !important 
 }
 :root {
-    --dt-secondary-container: ${#f1f3f4} !important;
+    --dt-secondary-container: #2f3335;
+}
+button[guidedhelpid="new_menu_button"]{ 
+    background-color: var(--dt-secondary-container)!important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5227]: What's the current background-color? We normally don't add CSS to then force it to use the original variables(we just fix the variable itself then).

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -17888,6 +17891,15 @@ nav a[href="/"] svg > g > path:first-child
 
 ================================
 
+urpredditodicittadinanza.lavoro.gov.it
+
+CSS
+:root{
+--lwc-colorContentAreaBackground : rgba(36, 39, 41, 0.98);
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17898]: Please use here as well the `${...}` form.

Comment (CarloHD) [reply src/config/dynamic-theme-fixes.config:17898]: ok, i've change it

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (CarloHD) [reply src/config/dynamic-theme-fixes.config:5227]: Without this fix the button have the same background-color of the whole page.
The bar on trash folder without this fix have a black background.

i'm trying to fix using the ${...} anyway. for now i've delete this part of modification

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -5221,7 +5221,10 @@ div[role="button"][aria-disabled="true"] > div {
     filter: invert(93.7%) hue-rotate(180deg) contrast(90.6%) !important 
 }
 :root {
-    --dt-secondary-container: ${#f1f3f4} !important;
+    --dt-secondary-container: #2f3335;
```
[the remark this answers is not in the document]
Comment (CarloHD) [reply src/config/dynamic-theme-fixes.config]: i've re-add it

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -5224,6 +5224,9 @@ div[role="button"][aria-disabled="true"] > div {
     --dt-secondary-container: ${#f1f3f4} !important;
 }
 
+button[guidedhelpid="new_menu_button"]{background-color: ${#dbdde0} !important;}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5227]:
```suggestion
button[guidedhelpid="new_menu_button"] {
    background-color: ${#dbdde0} !important;
}
```

File: src/config/dynamic-theme-fixes.config [hunk 5]
```diff
@@ -5224,6 +5224,9 @@ div[role="button"][aria-disabled="true"] > div {
...
+div[data-target="autopurgebanner"]>section {background: ${#dbdde0} !important;}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5228]:
```suggestion
div[data-target="autopurgebanner"] > section {
    background: ${#dbdde0} !important;
}
```

File: src/config/dynamic-theme-fixes.config [hunk 6]
```diff
@@ -17888,6 +17891,15 @@ nav a[href="/"] svg > g > path:first-child
 
 ================================
 
+urpredditodicittadinanza.lavoro.gov.it
+
+CSS
+:root{
+--lwc-colorContentAreaBackground : ${#ffffff};
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:17898]:
```suggestion
    --lwc-colorContentAreaBackground : ${white} !important;
```

**Review by Gusted — COMMENTED** [review]
Code-style

File: src/config/dynamic-theme-fixes.config (hunk 6 above)
Comment (CarloHD) [reply src/config/dynamic-theme-fixes.config:17898]: i've found better solution for the color

File: src/config/dynamic-theme-fixes.config (hunk 5 above)
Comment (CarloHD) [reply src/config/dynamic-theme-fixes.config]: done
anyway i've tried to have the same color of the search bar, but when i set the same color in `${...}` form, in dark mode it have another color, and not the expected color... 
in light mode the color of search bar is `f1f3f4`, in dark mode correctly change to `2f3335`; if i set `f1f3f4` to the banner, in dark mode becomes `101314`

**Review by Gusted — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config [hunk 7]
```diff
@@ -5224,6 +5224,13 @@ div[role="button"][aria-disabled="true"] > div {
     --dt-secondary-container: ${#f1f3f4} !important;
 }
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5226]:
```suggestion
```

**Pull request merged**